modules/rgbd/samples/kinfu_demo.cpp

```diff
@@ -208,7 +225,12 @@ int main(int argc, char **argv)
             {
                 imshow("depth", cvt8);
 
-                if(!kf->update(frame))
+                bool isCorr;
+                if (haveQS)
+                    isCorr = !kf->update(frame, pose);
+                else
+                    isCorr = !kf->update(frame, pose);
```

Comment:
both branches look identical - is it copy-paste issue?

Reply:
Fixed
I didn't notice, thx